libsolidity/interface/Natspec.cpp

```diff
@@ -168,7 +168,7 @@ Json::Value Natspec::devDocumentation(ContractDefinition const& _contractDef)
 			));
 	}
 
-	for (auto const& event: _contractDef.events())
+	for (auto const& event: _contractDef.definedInterfaceEvents())
```

Comment:
Not actually related to this PR directly, but I had to lookup the implementation of this function to understand what it does, as the name wasn't really intuitively telling me that.

Maybe (not in here) in the future we might want to give this function a better name. This function returns a list of all events directly or indirectly (via inheritance) declared in this contract. So maybe having a name like `collectAllVisibleInterfaceEvents()` might improve this.